packages/react-component-library/cypress/specs/ContextMenu/index.spec.ts

```diff
@@ -8,9 +8,11 @@ describe('ContextMenu', () => {
   describe('Storybook: Docs Mode', () => {
```

Comment:
It looks like this was an explicit test for the Docs tab (I guess we had a problem with the context menu being obscured there) – would be good to make the name consistent with the test in any case.

Reply:
Hmm yeah i'm not sure how that being docs tab specific would have been any different. I'll update the naming.